PendingReleaseNotes.md

```diff
@@ -21,6 +21,7 @@ ConfigMap can be used in mix with the already existing Env Vars defined in opera
 - Rook Ceph cleanupPolicy will clean up the dataDirHostPath only after user confirmation. For more info about CleanUpPolicy [read the design](https://github.com/rook/rook/blob/master/design/ceph/ceph-cluster-cleanup.md) as well as the documentation [cleanupPolicy](Documentation/ceph-cluster-crd.md#cluster-settings)
 - Rook monitor, mds and osd now have liveness probe checks on their respective sockets
 - Pools can now be configured to inline compress the data using the `compressionMode` parameter. Support added [here](https://github.com/rook/rook/pull/5124)
+- OSD on PVC now supports [topologySpreadConstraints](Documentation/ceph-cluster-crd.md#placement-configuration-settings).
```

Comment:
All the ceph daemons support the constraints now, right? They might not be useful for some daemons such as mons (should only have one per node anyway) or mgr (one one), but all other daemons can use the feature.

Reply:
True. I fixed the one-line description of this feature.